publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -492,8 +494,13 @@ public void shutdownCompleted(ShutdownSignalException cause) {
                 }
             });
             BasicProperties msgProps = MessageProperties.BASIC;
+            final BasicProperties PERSISTENT_BASIC_APPLICAION =
```

Comment:
I'd follow convetions given by MessageProperties and rename PERSISTENT_BASIC_APPLICATION -> PERSISTENT_BASIC_JSON or PERSISTENT_BASIC_APPLICATION_JSON.

Reply:
Renamed it